pandas/core/generic.py

```diff
@@ -3021,8 +3021,9 @@ def to_csv(
         ----------
         path_or_buf : str or file handle, default None
             File path or object, if None is provided the result is returned as
-            a string.  If a file object is passed it should be opened with
-            `newline=''`, disabling universal newlines.
+            a string.  If a non-binary file object is passed it should be opened
```

Comment:
can u add a version changed directive for 1.1

Reply:
added below the existing version change directive